2_process.yml

```diff
@@ -91,3 +92,14 @@ targets:
       coarse_points_ind='2_process/out/storm_points_coarse.rds.ind')
   2_process/out/storm_points_interp.rds:
     command: gd_get('2_process/out/storm_points_interp.rds.ind')
+
+  2_process/out/river_geoms.rds.ind:
+    command: process_river_geoms(
+      ind_file=target_name,
+      river_geoms_ind='1_fetch/out/river_geometry.rds.ind',
+      view_polygon=view_polygon,
+      coastal_threshold_sqkm=1000,
+      inland_threshold_sqkm=40000,
+      simplification_tolerance_m=5000)
```

Comment:
I think these thresholds and tolerances should go in the `viz_config.yml` where it will be easy to manipulate based based on the geographic area of the storm

Reply:
:+1: I like that distinction between configurable stuff that we'd want to build into the design for all vizzies (which wouldn't go in viz_config.yml) and configurable stuff that would vary by event (which would go in viz_config.yml). And I agree that these thresholds would make sense in that second category, though will defer to you @dblodgett-usgs if you disagree.

Reply:
This is in this location based on my naiveté only. I think that they should be in viz_config.yml. Will move.